app/assets/stylesheets/components/_banner.scss

```diff
@@ -0,0 +1,3 @@
+.gem-c-intervention {
```

Comment:
Hate to throw a spanner in the works, but we try to avoid styling components in applications as this breaks component isolation. Is there another approach that could be taken here? (maybe add a wrapper to the component that can be styled in the application?)

Reply:
Ah yep good idea thanks

Reply:
@andysellick is this ok?